ocpmodels/trainers/ocp_trainer.py

```diff
@@ -515,18 +520,27 @@ def predict(
                 return predictions
 
             ### Get unique system identifiers
-            sids = batch.sid.tolist()
+            sids = (
+                batch.sid.tolist()
+                if isinstance(batch.sid, torch.Tensor)
+                else list(batch.sid)
```

Comment:
If it isn't a tensor this will collate into a list so the `list()` call is redundant here.